travis/.rubocop_rspec_base.yml

```diff
@@ -95,9 +92,6 @@ PredicateName:
 Proc:
   Enabled: false
 
-RedundantReturn:
-  AllowMultipleReturnValues: true
```

Comment:
Possibly could have figured out a way to keep this, but just disabled it instead.